docs/install/project.md

```diff
@@ -52,16 +50,7 @@ As example, we will take [the tutorial project](https://github.com/leanprover-co
 
 * Go the the directory where you would like this package to live.
 
-* Run `git clone https://github.com/leanprover-community/tutorials.git`.
-
-* This creates a directory named `tutorials`. Enter it
-  with `cd tutorials`.
-
-* Type `leanpkg configure` to get `leanpkg` ready for use in this project.
-
-* Type `leanproject up` to get mathlib ready for use in this project.
-
-* Type `leanpkg build` to compile everything, this should only take a few seconds.
+* Run `leanproject get tutorials`
 
 * launch VScode, either through your application menu or by typing
   `code`
```

Comment:
This has nothing to do with leanproject but, for command line users, I think it's worth saying that `code folder_name` works.

Reply:
While we're at it we could also let macOS users know that they [need to take an extra step](https://code.visualstudio.com/docs/setup/mac#_launching-from-the-command-line) before they can run `code` from the command line. I could see this fitting either here or in the macOS instructions file (or both for extra redundancy).